lib/fetch/request.js

```diff
@@ -138,8 +138,47 @@ class Request {
 
     // 12. Set request to a new request with the following properties:
     request = makeRequest({
-      ...request,
-      window
+      // URL request’s URL.
+      // undici implementation note: this is set as the first item in request's urlList in makeRequest
+      // method request’s method. 
+      method: request.method,
+      // header list A copy of request’s header list.
+      headersList: request.headersList,
```

Comment:
Is this also cloned in makeRequest?

Reply:
it is https://github.com/nodejs/undici/blob/main/lib/fetch/request.js#L753-L755

Reply:
Maybe add a comment like you did below for urlList?